chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Announcement.java

```diff
@@ -20,99 +42,21 @@
     private Long lastModifiedUser;
 
     public Announcement() {
+        super();
+    }
+
+    public Announcement(AnnouncementEntity entity) {
```

Comment:
Do we still need this constructor AND the toDomain method on the entity? Todd prefers the former and I prefer the latter but we really only need 1 as far as I know.